src/components/dockerRegistry/Docker.tsx

```diff
@@ -509,6 +599,45 @@ const DockerForm = ({
         }))
     }
 
+    const handleOnChangeForRemoteConnectionRadio = (connectionType) => {
+        setRemoteConnectionMethod(connectionType)
+        setCustomState((_state) => ({
+            ..._state,
+            remoteConnectionConfig: {
+                ..._state.remoteConnectionConfig,
+                connectionMethod: { value: connectionType, error: '' },
+            },
+        }))
+    }
+
+    const handleOnChangeConfig = (e) => {
+        let { name, value } = e.target
```

Comment:
can we combined this and `updateWithCustomStateValidationForRemoteConnectionConfig`?

Reply:
yes we can, but then we would have to change the function definition for `onChange` in `CustomInput` (fe-common-lib), which would be a lot of task